bitrise.yml

```
@@ -634,3 +635,33 @@ workflows:
 
             cd tools/distribution && make
             venv/bin/python3 release.py "$GIT_TAG" --only-cocoapods
+
+  run_e2e_s8s_upload:
+    description: |-
+        Upload E2E application to Synthetics.
+    steps:
+    - script:
+        title: Upload E2E application to Synthetics.
+        inputs:
```

suggestion/ By now running `run_e2e_s8s_upload` as part of `push_to_develop_or_master` it means we will be updating E2E artifact also for new commits on `master`, which isn't always the right thing to do (think of a hotfix release while `develop` is much ahead of the latest release).

Let's narrow it to `develop` commits only, WDYT?
```suggestion
    - script:
        title: Upload E2E application to Synthetics.
        run_if: '{{enveq "BITRISE_GIT_BRANCH" "develop"}}'
        inputs:
```